### PR TITLE
Automated email to key account manager

### PR DESCRIPTION
**Pull request by Sachinbisht27**
Targets #2011 

### Description
Added Mail functionality in Project Module.

There are some projects surpassed end date but still marked as active. Sending mail to those project's Key Account manager.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Project/Resources/views/mail/ended-project.blade.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<div>
+	<style>
+		.line {
+			line-height: 1px;
+		}
+	</style>
+	<p>Hi,
+		@foreach ($projectData as $project) 
+	    @endforeach
```
Comment (gautam071) [inline Modules/Project/Resources/views/mail/ended-project.blade.php:9]: Why empty foreach?

File: Modules/Project/Resources/views/mail/ended-project.blade.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		{{$project['name']}}
+	</p>
+	 <p>There are some projects that have reached their end date but are still marked as active. We need you to take action so that the team can get clarity on active projects, the team's availability and assign other projects accordingly.</p>
+	 <table class="table">
+		<thead>
+			<tr>
+			<th scope="col">Project Name</th>
```
Comment (gautam071) [inline Modules/Project/Resources/views/mail/ended-project.blade.php:16]: Indentation

File: Modules/Project/Resources/views/mail/ended-project.blade.php (hunk 1 above)
Comment (gautam071) [inline Modules/Project/Resources/views/mail/ended-project.blade.php:9]: Indentation

File: Modules/Project/Resources/views/mail/ended-project.blade.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+					<td>
+						@foreach ($projectData as $project)
+						    <li><a href="{{ route('project.show', $project['project']) }}">{{$project['project']->name}}</a></li>
+						@endforeach
+					</td>
+				</tr>
+		</tbody>
+	 </table>
+	<br>
+    <p>To view all your projects, you can use this link  <a href="http://portal.test/projects">My Projects</a>.</p>
```
Comment (gautam071) [inline Modules/Project/Resources/views/mail/ended-project.blade.php:30]: Don't hard code the url

**Review by gautam071 — CHANGES_REQUESTED** [review]
Left some comments

File: Modules/Project/Resources/views/mail/ended-project.blade.php (hunk 1 above)
Comment (Sachinbisht27) [reply Modules/Project/Resources/views/mail/ended-project.blade.php:9]: I'll remove it.

File: Modules/Project/Resources/views/mail/ended-project.blade.php (hunk 2 above)
Comment (Sachinbisht27) [reply Modules/Project/Resources/views/mail/ended-project.blade.php:16]: ok

File: Modules/Project/Resources/views/mail/ended-project.blade.php (hunk 1 above)
Comment (Sachinbisht27) [reply Modules/Project/Resources/views/mail/ended-project.blade.php:9]: ok

File: Modules/Project/Resources/views/mail/ended-project.blade.php (hunk 3 above)
Comment (Sachinbisht27) [reply Modules/Project/Resources/views/mail/ended-project.blade.php:30]: ok

**Comment by Sachinbisht27** [comment]
@gautam071 Required Changes Done please review.

**Review by gautam071 — APPROVED** [review]
Looks good!

Can you just check wheater the blade file is formated?

**Pull request merged**